Parser/OperatingSystem.php

```diff
@@ -147,7 +148,7 @@ class OperatingSystem extends AbstractParser
         'GNU/Linux'             => [
             'LIN', 'ARL', 'DEB', 'KNO', 'MIN', 'UBT', 'KBT', 'XBT', 'LBT', 'FED',
             'RHT', 'VLN', 'MDR', 'GNT', 'SAB', 'SLW', 'SSE', 'CES', 'BTR', 'SAF',
-            'ORD', 'TOS', 'RSO', 'DEE', 'FRE',
+            'ORD', 'TOS', 'RSO', 'DEE', 'FRE', 'WHS',
```

Comment:
multiplatform Linux & Android  https://partner.zeasn.com/partnerportal/DeviceMapping/index.jhtml 
I think we should create a new OS family
```php
'Other Smart TV' => [
     'WHS'
]
```